llvm/test/CodeGen/PowerPC/sms-recmii.ll

```diff
@@ -0,0 +1,48 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc < %s -mtriple=powerpc64le-unknown-linux-gnu -verify-machineinstrs\
+; RUN:       -mcpu=pwr9 --ppc-enable-pipeliner --debug-only=pipeliner 2>&1 | FileCheck %s
+
```

Comment:
Thanks for adding the test.
-debug-only requires a build with assertions, so please add the following:
```
; REQUIRES: asserts
```